src/color-thief.js

```diff
@@ -119,7 +119,7 @@ ColorThief.prototype.getPalette = function(sourceImage, colorCount, quality) {
         a = pixels[offset + 3];
         // If pixel is mostly opaque and not white
         if (a >= 125) {
-            if (!(r > 250 && g > 250 && b > 250)) {
+            if (!(r > 255 && g > 255 && b > 255)) {
```

Comment:
RGB components are never > 255 ...

Reply:
Hi, 

Thanks for the contribution. While this change adresses the issue from the OP it also breaks the intended behavior referred to [earlier](https://github.com/lokesh/color-thief/pull/49#issuecomment-60825747) by @ksubileau. If you submit a PR which introduces a parameter so this can be turned off and on with appropriate defaults (ie. 'off' so people who update don't get unxepected behaviour) I'm sure someone would add appropriate [tests](https://github.com/lokesh/color-thief/blob/master/test/node-test.js) and update the [doc](https://github.com/lokesh/color-thief/blob/master/README.md) accordingly.